qutebrowser/browser/browsertab.py

```diff
@@ -1136,3 +1136,9 @@ def __repr__(self) -> str:
     def is_deleted(self) -> bool:
         assert self._widget is not None
         return sip.isdeleted(self._widget)
+
+    def is_in_background(self) -> bool:
+        active_tab = self.parent().currentWidget()
+        active_window = QApplication.instance().activeWindow()
+        return (self.tab_id != active_tab.tab_id
+                or self.win_id != active_window.win_id)
```

Comment:
`tab_id` seems to be global so I probably don't need to check for the win_id, right?